docs/howtos/adapt_vqe.rst

```diff
@@ -8,48 +8,62 @@ This tutorial outlines how the algorithm can be used.
 
 1. We obtain an ``ElectronicStructureProblem`` which we want to solve:
```

Comment:
This is just a comment - we can maybe improve them later (or now) - we should be able to have these all as class refs so they are clickable links not just highlighted. I see in places in the new howtos you do have class refs. Perhaps they should be used more generally, although some may be doable eg QubitMapper I think most should, and with intersphinx we can do it for VQE etc too right.